README.md

```diff
@@ -1,44 +1,78 @@
+  domain       = "example.com"
+  owners       = ["foo@example.com"]
+  managers     = ["example-sa@my-project.iam.gserviceaccount.com"]
+  members      = ["another-group@example.com"]
 }
 ```
 
-Functional examples are included in the
-[examples](./examples/) directory.
+Functional examples are included in the [examples](./examples/) directory.
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| bucket\_name | The name of the bucket to create | string | n/a | yes |
-| project\_id | The project ID to deploy to | string | n/a | yes |
+| customer\_id | Customer ID of the organization to create the group in. One of domain or customer_id must be specified | string | `""` | no |
+| description | Description of the group | string | `""` | no |
+| display\_name | Display name of the group | string | `""` | no |
+| domain | Domain of the organization to create the group in. One of domain or customer_id must be specified | string | `""` | no |
+| id | ID of the group. For Google-managed entities, the ID must be the email address the group | string | n/a | yes |
+| managers | Managers of the group. Each entry is the ID of an entity. For Google-managed entities, the ID must be the email address of an existing group, user or service account | list | `<list>` | no |
+| members | Members of the group. Each entry is the ID of an entity. For Google-managed entities, the ID must be the email address of an existing group, user or service account | list | `<list>` | no |
+| owners | Owners of the group. Each entry is the ID of an entity. For Google-managed entities, the ID must be the email address of an existing group, user or service account | list | `<list>` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| bucket\_name |  |
+| id | ID of the group. For Google-managed entities, the ID is the email address the group |
+| resource\_name | Resource name of the group in the format: groups/{group_id}, where group_id is the unique ID assigned to the group. |
 
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 
+## Limitations
+
+The provider is still under development, the following are known issues or
+limitations:
+
+* Updating a `google_cloud_identity_group_membership` to remove a role fails
+    with an error
+    ([link](https://github.com/hashicorp/terraform-provider-google/issues/7616)).
+
+* [InitialGroupConfig](https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups/create#initialgroupconfig)
+    is not supported in the `google_cloud_identity_group` resource, which
+    prevents non-admins from creating groups in an organization, even when the
+    organization is configured to allow anyone to create groups.
+
+* Only
+    [Google Groups](https://cloud.google.com/identity/docs/groups#group_properties)
+    are supported.
+
+* Last `OWNER` cannot be removed from a Google Group.
```

Comment:
This may cause issues during CI tests with the current [example](https://github.com/terraform-google-modules/terraform-google-group/blob/40dd5ceccdb1ef22d9873e8e57e678828937779e/examples/simple_example/main.tf) as owner is an SA created in same config.

Reply:
Good catch. I removed assigning owners to the group. I tried it locally, and the group can be successfully destroyed with out any owners in it.